README.md

```diff
@@ -59,6 +60,7 @@ If you set it to an empty string, `grade` will print line protocol to stdout.
 * The Go version subcommand will produce a string like `go1.6.2 darwin/amd64`, but you can use any string you'd like.
 * The revision subcommand is the full SHA of the commit, but feel free to use a git tag name or any other string.
 * The timestamp is a Unix epoch timestamp in seconds.
+* The branch subcommand is the name of the current branch.
```

Comment:
Please clarify that the branch is optional.

Reply:
done